commands/admin/log/log.js

```
@@ -215,10 +221,31 @@ module.exports = {
-          })
+          try {
+            const fetched = await interaction.guild.channels.fetch(channelId)
+            if (!fetched || !fetched.isTextBased()) {
+              return interaction.reply({
+                content:
+                  '❌ The configured log channel could not be found or is not a text channel. Please set it again.',
+                flags: MessageFlags.Ephemeral,
+              })
+            }
+            await fetched.send({ embeds: [embed] })
+            return interaction.reply({
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)